.changeset/red-drinks-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+"@skeletonlabs/skeleton": minor
+---
+
+bugfix: ProgressBar component classesMeter applied twice in the template #2280
```

Comment:
Hey @ishaanparlikar this is nearly perfect, only one thing - can you strip out the Issue link at the end of the changeset description. When the changeset is ingested on deployment it will automatically insert this. Meaning we'll have duplication.

Once that's done I'm happy to merge. Thanks!

Reply:
Sure. So should I submit a new changeset?
Forgive me for my noobness. First time working on this type of deployment structure.

Reply:
Hey @endigo9740 
Can you check now?

Reply:
Perfect, thanks!